test/telemetry-datadog.test.js

```diff
@@ -0,0 +1,9 @@
+const Seneca = require('seneca')
+
+describe('github-provider', () => {
```

Comment:
Please give the test a more appropriate description, e.g. `describe('automated tests')` and `test('works')`.